### PR TITLE
[PWGJE] Add workflows for charged jet spectra analysis

### PR DESCRIPTION
**Pull request by wefeng1110**
Add workflows for jet spectra, background study:

1. PWGJE/Tasks/jetSpectraCharged.cxx
2. PWGJE/Tasks/jetBackgroundAnalysis.cxx

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,90 +0,0 @@
-# Copyright 2019-2020 CERN and copyright holders of ALICE O2.
```
Comment (nzardosh) [inline CMakeLists.txt:1]: what is this file and why was it removed?

File: PWGJE/Tasks/jetMatchedResponse.cxx [hunk 2]
```diff
@@ -0,0 +1,392 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
```
Comment (nzardosh) [inline PWGJE/Tasks/jetMatchedResponse.cxx:2]: why is this in a separate task? You can combine it into the same task as the jetSpectraCharged. It is better not to separate the MC and data parts of tasks into multiple files

File: PWGJE/Tasks/jetSpectraCharged.cxx [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+      registry.fill(HIST("jet_observables_eventwiseconstituentsubtracted"), jet.pt(), jet.eta(), jet.phi(), weight);
+    }
+  }
+
+  template <typename TTracks>
+  void fillTrackHistograms(TTracks const& track, float weight = 1.0)
+  {
+      registry.fill(HIST("h_track_pt"), track.pt(), weight);
+      registry.fill(HIST("h2_track_eta_track_phi"), track.eta(), track.phi(), weight);
+      registry.fill(HIST("h2_track_pt_track_dcaxy"), track.pt(), track.dcaXY(), weight);
```
Comment (nzardosh) [inline PWGJE/Tasks/jetSpectraCharged.cxx:205]: this task seems fine but it seems to be a copy of the jetFinderQA in large parts. That is fine to some extent in this case but I am wondering if all of these types of histograms are needed?

File: CMakeLists.txt (hunk 1 above)
Comment (wefeng1110) [reply CMakeLists.txt:1]: Sorry, I delete this CMakeList.txt by mistake. I will upload it.

File: PWGJE/Tasks/jetMatchedResponse.cxx (hunk 2 above)
Comment (wefeng1110) [reply PWGJE/Tasks/jetMatchedResponse.cxx]: Hi Nima, I just combine this matched task with the jetSpectraCharged. but I would like to keep several TH1 and TH2 for track and raw jet(before subtraction) to just confirm i was doing the right cut.

**Comment by nzardosh** [comment]
Hi @wefeng1110 this PR cannot be approved until the formatting test is passed in github since it is required

**Comment by wefeng1110** [comment]
Hi @nzardosh thank you very much for your reminder, so far my PR has passed all the format tests.

**Review by nzardosh — APPROVED** [review]

**Pull request merged**